docs-src/references/web-ui-configuration.md

```diff
@@ -204,3 +158,89 @@ Prevents the user from resetting Workflows from the Web UI.
 ```yaml
 workflowResetDisabled: false
 ```
+
+## batchActionsDisabled
+
+Prevents the execution of Batch actions.
+
+```yaml
+batchActionsDisabled: false
+```
+
+## hideWorkflowQueryErrors
+
+Hides any errors resulting from a Query to the Workflow.
+
+```yaml
+hideWorkflowQueryErrors: false
+```
+
+## cors
+
+The name of the `cors` field stands for Cross-Origin Resource Sharing.
+Use this field to provide a list of domains that are authorized to access the UI Server APIs.
+
+```yaml
+cors:
+  cookieInsecure: false
+  allowOrigins:
+    - http://localhost:3000 # used at development by https://github.com/temporalio/ui
+```
+
+## tls
+
+Transport Layer Security (TLS) configuration.
+
+```yaml
+tls:
+  caFile:
+  certFile:
+  keyFile:
+  caData:
+  certData:
+  keyData:
+  enableHostVerification: false
+  serverName:
+```
+
+## auth
+
+Auth configuration.
```

Comment:
Do we have more information about this somewhere in docs? I'd like to know what these fields in the YAML mean. What are the possible providers people can use, etc.

Reply:
Not in docs. I can ask the engineers.